<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exit_mezcla_window</class>
 <widget class="QWidget" name="exit_mezcla_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Salida mezcla</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="central_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="backgroundcolor_frame">
        <property name="styleSheet">
         <string notr="true">background-color: #537188;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="content_frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QComboBox" name="productos_para_mezcla">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>140</y>
              <width>211</width>
              <height>51</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
    background-color: #E1D4BB;
    color: #537188;
    border: 2px solid #CBB279;
    border-radius: 5px;
    padding: 5px;
    margin: 5px;
    font-size: 14px;
}

QComboBox::drop-down {
    border: none;
}

QComboBox::down-arrow {
    image: url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-expand-arrow-24.png);
}

QComboBox QAbstractItemView {
    border: 2px solid #537188;
    selection-background-color: #CBB279;
}</string>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>100</y>
              <width>211</width>
              <height>43</height>
             </rect>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
    color: #ffffff;
    border: 2px solid #537188;
    border-radius: 5px;
    padding: 5px;
    margin: 5px;
    font-size: 14px;
    text-align: center;
    background-color: #CBB279;
}</string>
            </property>
            <property name="text">
             <string>INGREDIENTE</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="cantidad_exit_paramezcla">
            <property name="geometry">
             <rect>
              <x>220</x>
              <y>140</y>
              <width>101</width>
              <height>51</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QDoubleSpinBox {
    background-color: #E1D4BB;
    color: #537188;
    border: 2px solid #CBB279;
    border-radius: 5px;
    padding: 5px;
    margin: 5px;
    font-size: 14px;
}

QDoubleSpinBox::up-button {
    width: 20px;
    border: none;
    background: #537188 url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-collapse-arrow-24.png) no-repeat center;
}

QDoubleSpinBox::down-button {
    width: 20px;
    border: none;
    background: #537188 url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-expand-arrow-24.png) no-repeat center;
}

QDoubleSpinBox::up-button:pressed {
    background: #CBB279 url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-collapse-arrow-24.png) no-repeat center;
}

QDoubleSpinBox::down-button:pressed {
    background: #CBB279 url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-expand-arrow-24.png) no-repeat center;
}</string>
            </property>
            <property name="maximum">
             <double>9999.989999999999782</double>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>220</x>
              <y>100</y>
              <width>101</width>
              <height>43</height>
             </rect>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
    color: #ffffff;
    border: 2px solid #537188;
    border-radius: 5px;
    padding: 5px;
    margin: 5px;
    font-size: 14px;
    text-align: center;
    background-color: #CBB279;
}</string>
            </property>
            <property name="text">
             <string>CANTIDAD</string>
            </property>
           </widget>
           <widget class="QPushButton" name="submit_exit_mezcla">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>420</y>
              <width>131</width>
              <height>51</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #E1D4BB;
    color: #537188;
    border: 2px solid #CBB279;
    border-radius: 5px;
    padding: 5px;
    margin: 5px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #CBB279;
    border: 2px solid #537188;
}

QPushButton:pressed {
    background-color: #CBB279;
    border: 2px solid #537188;
    padding-left: 7px;
}</string>
            </property>
            <property name="text">
             <string>Aplicar</string>
            </property>
           </widget>
           <widget class="QTableWidget" name="ingredientes_mezcla">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>230</y>
              <width>311</width>
              <height>192</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QTableWidget {
    background-color: #E1D4BB;
    color: #537188;
    border: 2px solid #CBB279;
    border-radius: 5px;
    margin: 5px;
    font-size: 14px;
    gridline-color: #537188;
}

QTableWidget::item {
    border: 1px solid #537188;
}

QTableWidget::item:selected {
    background-color: #CBB279;
    color: #E1D4BB;
}

QHeaderView::section {
    background-color: #E1D4BB;
    color: #537188;
    padding: 5px;
    border: 2px solid #CBB279;
    border-radius: 5px;
    margin: 5px;
    font-size: 14px;
}</string>
            </property>
            <column>
             <property name="text">
              <string>Ingrediente</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Cantidad</string>
             </property>
            </column>
           </widget>
           <widget class="QComboBox" name="mezcla_combobox">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>211</width>
              <height>51</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
    background-color: #E1D4BB;
    color: #537188;
    border: 2px solid #CBB279;
    border-radius: 5px;
    padding: 5px;
    margin: 5px;
    font-size: 14px;
}

QComboBox::drop-down {
    border: none;
}

QComboBox::down-arrow {
    image: url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-expand-arrow-24.png);
}

QComboBox QAbstractItemView {
    border: 2px solid #537188;
    selection-background-color: #CBB279;
}</string>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>mezclas</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>211</width>
              <height>43</height>
             </rect>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
    color: #ffffff;
    border: 2px solid #537188;
    border-radius: 5px;
    padding: 5px;
    margin: 5px;
    font-size: 14px;
    text-align: center;
    background-color: #CBB279;
}</string>
            </property>
            <property name="text">
             <string>MEZCLA</string>
            </property>
           </widget>
           <widget class="QPushButton" name="agregar_ingrediente">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>190</y>
              <width>311</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #E1D4BB;
    color: #537188;
    border: 2px solid #CBB279;
    border-radius: 5px;
    padding: 5px;
    margin: 5px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #CBB279;
    border: 2px solid #537188;
}

QPushButton:pressed {
    background-color: #CBB279;
    border: 2px solid #537188;
    padding-left: 7px;
}</string>
            </property>
            <property name="text">
             <string>Agregar</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>220</x>
              <y>10</y>
              <width>101</width>
              <height>43</height>
             </rect>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
    color: #ffffff;
    border: 2px solid #537188;
    border-radius: 5px;
    padding: 5px;
    margin: 5px;
    font-size: 14px;
    text-align: center;
    background-color: #CBB279;
}</string>
            </property>
            <property name="text">
             <string>CANTIDAD</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="cantidad_mezcla">
            <property name="geometry">
             <rect>
              <x>220</x>
              <y>50</y>
              <width>101</width>
              <height>51</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QDoubleSpinBox {
    background-color: #E1D4BB;
    color: #537188;
    border: 2px solid #CBB279;
    border-radius: 5px;
    padding: 5px;
    margin: 5px;
    font-size: 14px;
}

QDoubleSpinBox::up-button {
    width: 20px;
    border: none;
    background: #537188 url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-collapse-arrow-24.png) no-repeat center;
}

QDoubleSpinBox::down-button {
    width: 20px;
    border: none;
    background: #537188 url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-expand-arrow-24.png) no-repeat center;
}

QDoubleSpinBox::up-button:pressed {
    background: #CBB279 url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-collapse-arrow-24.png) no-repeat center;
}

QDoubleSpinBox::down-button:pressed {
    background: #CBB279 url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-expand-arrow-24.png) no-repeat center;
}</string>
            </property>
            <property name="maximum">
             <double>9999.989999999999782</double>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
